packages/amplication-client/src/UserAction/useUserActionWatchStatus.ts

```diff
@@ -1,9 +1,9 @@
 import { useContext, useEffect } from "react";
 import { useQuery } from "@apollo/client";
 
-import * as models from "../../models";
-import { GET_USER_ACTION } from "./queries";
-import { AppContext } from "../../context/appContext";
+import * as models from "../models";
+import { GET_USER_ACTION } from "../Entity/ImportPrismaSchema/queries";
```

Comment:
Shouldn't the GET_USER_ACTION query also move out of /Entity/ImportPrismaSchema?